examples/esm/chat/js/index_chat.js

```diff
@@ -9,11 +9,12 @@ chatWindowInstance.installPlugin(new AgentDesktopPlugin());
 //OPTION #1
 let botOptions=chatConfig.botOptions;
 
-botOptions.JWTUrl = "PLEASE_ENTER_JWTURL_HERE";
-botOptions.userIdentity = 'PLEASE_ENTER_USER_EMAIL_ID';// Provide users email id here
-botOptions.botInfo = { name: "PLEASE_ENTER_BOT_NAME", "_id": "PLEASE_ENTER_BOT_ID" }; // bot name is case sensitive
-botOptions.clientId = "PLEASE_ENTER_CLIENT_ID";
-botOptions.clientSecret = "PLEASE_ENTER_CLIENT_SECRET";
+botOptions.koreAPIUrl = "https://bots.kore.ai/api";
```

Comment:
@KambagiriswamyBo-Kore  remove configs

and validate your agent desktop changes in following two files

examples/esm/chat-with-plugins/js/index_chat.js
examples/umd/chat-with-plugins/index.html

Reply:
@RajasekharBa-Kore Removed the configs